ct.yaml

```diff
@@ -1,7 +1,5 @@
 # See https://github.com/helm/chart-testing#configuration
-remote: origin
 chart-dirs:
   - charts
-chart-repos:
-  - incubator=https://kubernetes-charts-incubator.storage.googleapis.com
-helm-extra-args: --timeout=500s
+remote: origin
+target-branch: main
```

Comment:
The target-branch changes from _master_ to _main_.

Reply:
Why are LNs 6 and 7 not needed?

Reply:
Nvm about LN 7, I see the comment just below.

Reply:
For LN 6, the comment is at above.